static/bidder-info/iionads.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://{{.Host}}/openrtb/{{.PublisherID}}"
+maintainer:
+  email: "engineering@project-limelight.com"
+capabilities:
+  app:
+    mediaTypes:
```

Comment:
The mediaTypes mentioned in the [doc PR](https://github.com/prebid/prebid.github.io/pull/4297/files) are not matching. Could you please update the doc PR to reflect all the supported media types.

Reply:
As noted in the docs PR https://github.com/prebid/prebid.github.io/pull/4297, we have to use the same documentation with our Prebid.js adapter. This means that we must specify in the documentation only the set of media types that both our Prebid.js and Prebid Server adapters support.

Reply:
It would be fine to add a note to your docs to explain this. e.g. "the client-side adapter doesn't support X, but the server-side adapter does."

Reply:
@bretg Thank you for your comment. I added this note in https://github.com/prebid/prebid.github.io/pull/4297/commits/d7223cf3e374f2e9a4e085283b63fcfe944c34a8 commit